src/main/java/seedu/duke/constants/ErrorMessage.java

```diff
@@ -10,10 +10,16 @@ public interface ErrorMessage {
     String INVALID_NUMERICAL_AMOUNT = "Please provide a float amount to update";
     String MORE_THAN_ONE_CURRENCY_PROVIDED = "Please do not provide more than one currency.";
     String NO_SUCH_ACCOUNT = "You do not have an account for the currency.";
+
+    String INVALID_WITHDRAW_COMMAND = "Please check that you have correctly provided the currency and amount";
+    String INSUFFICIENT_WITHDRAW_BALANCE = "You do not have sufficient balance to withdraw.";
+    String NEGATIVE_WITHDRAWAL_AMOUNT   = "You have entered a negative withdrawal amount.";
```

Comment:
I think NEGATIVE_WITHDRAWAL_AMOUNT is a bit too specific of an exception. I believe it would already be categorised as INVALID_WITHDRAW_COMMAND